client/hotrod-client/src/test/java/org/infinispan/client/hotrod/retry/DistributionRetryTest.java

```diff
@@ -114,14 +116,14 @@ private Object generateKeyAndShutdownServer() throws IOException, ClassNotFoundE
 
       remoteCache.put(key, "v");
       assertOnlyServerHit(getAddress(hotRodServer2));
-      TcpTransportFactory tcpTp = (TcpTransportFactory) ((InternalRemoteCacheManager) remoteCacheManager).getTransportFactory();
+      ChannelFactory channelFactory = ((InternalRemoteCacheManager) remoteCacheManager).getChannelFactory();
```

Comment:
Looking how `ChannelFactory` is being used I think it may be beneficial to call it Netty (or Channel if you wish) Transport Factory.

Reply:
It's not creating any `Transport` instances, therefore Transport Factory does not match. NettyChannelFactory would be redundant.